src/components/_helpers/resolveContextOrProp.js

```diff
@@ -1,4 +1,8 @@
 export const resolveContextOrProp = (contextValue, propValue) => {
+  if (contextValue === false) {
```

Comment:
Could you please rewrite it so there are only two returns?

```
if (…) {
  return a;
}

return b;
```

Having three returns makes it hard to follow the flow of the code. I know this function is short, but still…